### ISSUE TITLE
Context needed on submission decisions.

### ISSUE THREAD

**Issue by nodunayo** [nodunayo/speakerline#7]
Proposals get accepted and rejected for many reasons. For a site like Speakerline, being able to add context to rejections is particularly important: is it that there were clear flaws in the structure of the submission or was the proposal excellent but it didn't suit the conference? Did the speaker already get a talk accepted at the same conference?

What ideas do you have for how we can add more context to the submissions?

**Comment by jenncoop** [comment]
Note from our chat at RailsConf: we discussed that we would start by allowing folks to add a note to `submissions/<id>/edit` that would show if a "rejected" status is selected.

**Comment by nodunayo** [comment]
@jenncoop Yes! I reckon we should have the option when a submission is created as well? What do you think?

**Comment by jenncoop** [comment]
@nodunayo Yes, absolutely, good point, I'll include the changes when creating a submission as well.
